### DISCUSSION TITLE
Reading & writing nested data to Parquet

### DISCUSSION THREAD

**Discussion by hohav** [jorgecarleitao/arrow2#647] [Q&A] [answered]
The README says that reading & writing of structs and nested lists to Parquet is not supported, but is this a TODO or a non-goal? I believe it's the former due to [this discussion](https://github.com/jorgecarleitao/parquet2/discussions/59#discussioncomment-1459759), and I see #504 for reading structs, but I don't see an issue for writing them.

BTW, should #504 have been closed by #547 or am I misreading one of them?

**Comment by jorgecarleitao** [comment, marked as answer]
You are right on all accounts; it is a todo; implementing dremmel sucks a bit of my soul, so I take it in small chunks.

**Reply by hohav** [reply]
Awesome, thank you. Are there any pieces that someone without deep knowledge of this project could contribute? I'd love to help out if I can.

**Reply by jorgecarleitao** [reply]
Awesome 🙇 . The issues as good first issues https://github.com/jorgecarleitao/arrow2/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22 are a good starting point.

A simpler activity is to move tests from the `src/` to `tests/it/`, so that we do not need to re-compile the crate to add/change tests.
